README.md

```diff
@@ -24,9 +24,39 @@ sharing of data, prevents the "bitrot" of datasets wherein details of the
 capture are lost over time, and makes it possible for different tools to operate
 on the same dataset, thus enabling data portability between tools and workflows.
 
-(Taken from the
-[Introduction](https://github.com/gnuradio/SigMF/blob/master/sigmf-spec.md#introduction)
-of the specification document.)
+SigMF signal recordings typically involve a data file (e.g. a binary file of IQ
+or RF samples) and a metadata file containing plain text that describes the data.
+Together these files represent one recording, such as `example.sigmf-data` and
+`example.sigmf-meta`.  Here is a minimal example of a SigMF `.sigmf-meta` file:
+
+```yaml
+{
+    "global": {
+        "core:datatype": "cf32_le"
+        "core:sample_rate": 1000000,
+        "core:hw": "PlutoSDR with 915 MHz whip antenna",
+        "core:author": "Art Vandelay",
+        "core:version": "1.0.0",
+    },
+    "captures": [
+        {
+            "core:sample_start": 0
+            "core:frequency": 915000000,
+        }
+    ],
+    "annotations": []
+}
+```
+
+## Using SigMF
+
+There are at least four ways you can use SigMF today, thanks to the community-supported projects:
+
+1. Within **Python**, using the Python package included in this repo and discussed below (avoid installing with pip for the time being, 1.0.0 isn't on PyPI yet)
```

Comment:
@bhilburn what do you think about this note?  Do you also want a banner at the top of the readme?

Reply:
I suggest that the note about `pip` be relocated closer to line 83.  Also, aside from the timing of sigmf arriving on PyPI, a `.deb` is a 90% solution, which is pretty easily generated using this command, and could be included with the release files:
```bash
fpm -s python -t deb --python-bin /usr/bin/python3 --python-package-name-prefix python3 setup.py
```

Reply:
Hold on, let me see about the pip thing

Reply:
Okay, pip is now up-to-date with v1.0.0 👍🏼